### ISSUE TITLE
Add Preload and Resource Hints to status dashboard

### ISSUE THREAD

**Issue by igrigorik** [MicrosoftEdge/Status#183]
Spec: https://w3c.github.io/preload/

> This specification defines the preload relationship of the HTML Link Element (`<link>`).

Spec: http://w3c.github.io/resource-hints/

> This specification defines the preconnect relationship and extends the preload relationship of the HTML Link Element (`<link>`). These primitives enable the developer, and the server generating or delivering the resources, to assist the user agent in the decision process of which origins it should connect to, and which resources it should fetch to improve page performance.

---

**Blink intent to implement and ship** for `<link rel=preconnect>` (part of Resource Hints): https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/CM5BaP6uVvE

Support for other parts of both specs in Blink will follow / will be done via separate intents.

/cc @tobint @pmeenan @dstorey

**Comment by jacobrossi** [comment]
Sorry I have been unreasonably slow to respond to this.  I noticed only the [preconnect portion](https://www.chromestatus.com/features/5560623895150592) is listed on ChromeStatus.  Would it be possible for you to add these other features there first?  

That way status.modern.ie can pull data from ChromeStatus, which will help us keep the Chrome implementation status up to date on our site.

**Comment by igrigorik** [comment]
@jacobrossi updated preconnect page with latest status.

@yoavweiss we should send an "intent to implement" for preload, since you're already making progress on it... Do you mind creating the entry on Chromium dashboard? :)

**Comment by yoavweiss** [comment]
Yeah, sure. I'll create the entry and send out an "intent to implement".

**Comment by igrigorik** [comment]
@yoavweiss thanks!

**Comment by yoavweiss** [comment]
Filed https://www.chromestatus.com/features#preload

Still need to fill it in with better details and get these OWP bug and intent to implement in place :)

@igrigorik - did @mcmanus start work/showed public support on "preload" as well?

**Comment by igrigorik** [comment]
@yoavweiss afaik, no.. but I'm hoping he will! I'll defer to him though... ;-)

**Comment by toddreifsteck** [comment]
Talked with @jacobrossi . This is now my task. @toddreifsteck

**Comment by toddreifsteck** [comment]
2 years later.. and I'll submit a PR later today.

**Comment by igrigorik** [comment]
![yay](https://media.tenor.com/images/5c90bc443c6af11e5d9c57244fa661be/tenor.gif)
